fabric-transfer-api-v1/src/main/java/net/fabricmc/fabric/api/transfer/v1/storage/base/ResourceAmount.java

```diff
@@ -16,6 +16,8 @@
 
 package net.fabricmc.fabric.api.transfer.v1.storage.base;
 
+//CHECKSTYLE.OFF: MatchXpath
```

Comment:
These `MatchXpath` checks could have different IDs so that they can be suppressed individually

Reply:
Im not aware of a way to do this

Reply:
https://checkstyle.sourceforge.io/config.html#Id

Reply:
I tried that, it doesnt seem to work.

<img width="757" alt="Screenshot 2024-12-06 at 10 38 49" src="https://github.com/user-attachments/assets/64e6a13d-e8bd-45d3-aab1-8aa93832a5f9">

I also cannot get the custom messages to work.

Reply:
Oh well, I guess leave it then